chapter_13_database_layer_validation.asciidoc

```diff
@@ -578,8 +592,47 @@ $ *git commit -am "Adjust new list view to do model validation"*
 ----
 
 
+TIP: When working on a new feature, it's common to realize partway through that
+    a refactor of the application is needed. Adding an early return to the FT
+    you're currently working on allows you to perform this refactor against
+    passing FTs, even while the feature is still in progress.
```

Comment:
let's move this to the end of the section.  we normally have TIPs _after_ the example of the thing we're suggesting.